understanding/20/keyboard-no-exception.html

```diff
@@ -27,8 +27,35 @@ <h2>Intent of Keyboard (No Exception)</h2>
          function requires input that depends on the path of the user's movement and not just
          the endpoints (excluded from the requirements of 2.1.1) must be made keyboard accessible.
          Rather, it means that content that uses path-dependent input cannot conform to this
-         Success Criterion and therefore cannot meet Guideline 2.1 at Level AAA. 
+         Success Criterion and therefore cannot meet Guideline 2.1 at Level AAA.
       </p>
+
+      <div class="note">
+         <p>Platforms and user agents usually have conventions for how web content or
+            applications are controlled with a keyboard interface. If content does not follow
+            the platform/user agent conventions it may be difficult to use, as users will need
+            to learn different interaction methods. As a <em>best practice</em>, content
+            should follow the platform/user agent conventions. However, deviating from these
+            conventions does <em>not</em> fail the normative requirement of this Success Criterion.</p>
+         <p>
+            For instance, buttons that have focus can generally be activated using both the
+            <kbd>Enter</kbd> key and the <kbd>Space</kbd> bar. If a custom button control
+            in a web application instead only reacts to <kbd>Enter</kbd>
+            (or even a completely custom key or key combination), this still
+            <strong>passes</strong> the requirements of this Success Criterion.
```

Comment:
Nitpick; Success criteria aren't passed, they are satisfied.

```suggestion
            <strong>satisfy</strong> the requirements of this Success Criterion.
```

Reply:
weirdly (but not surprisingly) WCAG's various understandings and techniques seem to use both "passes" and "satisfies" interchangeably (i.e. there are quite a few cases of "passes"/"fails")

Reply:
the correct replacement here would then be "satisfies" (rather than "satisfy")

Reply:
resolved via https://github.com/w3c/wcag/pull/1642/commits/38cce8aada87e526bc2fe0e1449b073828672819